lua/nvim-treesitter-playground/internal.lua

```diff
@@ -35,9 +42,18 @@ local playground_ns = api.nvim_create_namespace('nvim-treesitter-playground')
 local query_hl_ns = api.nvim_create_namespace('nvim-treesitter-playground-query')
 
 local function get_node_at_cursor()
-  local success, node_at_point = pcall(function() return ts_utils.get_node_at_cursor() end)
+  local lnum, col = unpack(vim.api.nvim_win_get_cursor(0))
+  local root_lang_tree = parsers.get_parser()
+  local owning_lang_tree = root_lang_tree:language_for_range({lnum - 1, col, lnum - 1, col})
+  local result
 
-  return success and node_at_point or nil
+  for _, tree in ipairs(owning_lang_tree:trees()) do
+    result = tree:root():named_descendant_for_range(lnum - 1, col, lnum - 1, col)
+
+    if result then break end
```

Comment:
you could return result instead of breaking, but i guess it doesn't matter too much :)